### DISCUSSION TITLE
Remove the sidebar, navigation bar, and toc in a MyST site

### DISCUSSION THREAD

**Discussion by khlee42** [executablebooks/meta#1064] [Q&A]
I like MyST features, but for my specific use case, I'd like each page to function as a standalone webpage by removing these navigation features. Is it possible to implement this?

**Comment by rowanc1** [comment]
We are currently working on the article theme which may be more to your liking?

[![article-theme](https://github.com/executablebooks/meta/assets/913249/f8abe4f2-e772-4e10-8f2c-8ea3a6af6188)](http://simpeg.xyz/tle-finitevolume/)


To switch to that theme:

```yaml
site:
    template: article-theme
```

The one above has a project banner and multiple supporting notebooks. ([source](https://github.com/simpeg/tle-finitevolume), [example](https://simpeg.xyz/tle-finitevolume/)).

On the book theme there are also the following options:

```yaml
site:
  hide_toc: true
  hide_footer_links: true
  hide_outline: true
```

**Reply by khlee42** [reply]
Thanks for the quick response! I tried out `article-theme`, and it seems only `hide_footer_links` option works. The other two, `hide_toc` and `hide_outline`, changed nothing in the page.

**Reply by rowanc1** [reply]
Yes, those options were for the default `book-theme`. We should probably implement the `hide_outline` in the article theme though!

**Reply by khlee42** [reply]
I thought those options were implemented for the `article-theme`. It worked perfectly in the `book-theme` as you suggested. It'd be nice to have this in the documentation somewhere. Thank you so much!
